### ISSUE TITLE
[API Proposal]: Arm Neon Intrinsics - float AddAcross is missing, eg vaddvq_f32 

### ISSUE THREAD

**Issue by BmanClark** [dotnet/runtime#58547]
### Background and motivation

Fill out missing section of API to be more full - add ability for AddAcross to be same for float as others.

### API Proposal

```C#
namespace System.Runtime.Intrinsics.Arm.AdvSimd.Arm64
{
     public static Vector64<float> AddAcross(Vector128<float> value);
}
```     

### API Usage

```C#
// add a float vectors components to a single value
float addsUp = System.Runtime.Intrinsics.Arm.AdvSimd.Arm64.AddAcross(myFloatVector).ToScalar();


### Risks

_No response_

**Comment by BmanClark** [comment]
@TamarChristinaArm & @tannergooding - I've raised a ticket for this missing function and those close to it... (as I've mentioned it to both of you already...)

**Comment by TamarChristinaArm** [comment]
Thanks!, Arrgg, I realized after I asked you to open the issue, looking at it again though, I don't think it's a bug.

During API design we decided not to implement pseudo-intrinsics. the `addv` instruction on NEON only supports integer values. in C we map the floating point ones to `faddp`. so this case for instance is https://developer.arm.com/architectures/instruction-sets/intrinsics/vaddvq_f32 which is mapped to two `faddp` calls.

So don't think it's actually a bug, just a different API philosophy.  The easiest probably none confusing use is using the `.Sum ()` API? https://github.com/dotnet/runtime/issues/35626
